roles/ipaclient/vars/RedHat-7.yml

```diff
@@ -2,4 +2,4 @@
 # vars/RedHat-7
 ---
 ipaclient_packages: [ "ipa-client", "libselinux-python" ]
-#ansible_python_interpreter: '/usr/bin/python2'
+# ansible_python_interpreter: '/usr/bin/python2'
```

Comment:
I think the ansible_python_interpreter lines can be removed. This is handled by Ansible.

Reply:
Fixed.